tsinfer/progress.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         augment_ancestors=False,
         match_samples=False,
         verify=False,
+        **kwargs,
```

Comment:
Maybe tqdm_kwargs would be more explanatory?

Reply:
Yeah good idea - fixed.